src/app/clients/clients.service.ts

```diff
@@ -37,8 +37,12 @@ export class ClientsService {
     return this.http.get('/clients', { params: httpParams });
   }
 
-  getClientTemplate(): Observable<any> {
-    return this.http.get('/clients/template');
+  getClientTemplate(officeId: number): Observable<any> {
```

Comment:
It would be nicer if there would be 2 method. one with param and one without it, no?

Reply:
Done! Updated